.github/workflows/linux-build-gcc.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
             -DCMAKE_CXX_COMPILER_LAUNCHER=sccache
 
       - name: Build Project
-        run: cmake --build build --parallel $((`nproc`)) -- -w dupbuild=warn
```

Comment:
https://github.com/ninja-build/ninja/pull/2356

this flag is no longer supported as of 1.12.0, looks like we're using 1.12.1 now